lib/Doctrine/Common/Cache/FileCache.php

```diff
@@ -124,11 +112,26 @@ public function getExtension()
      */
     protected function getFilename($id)
     {
+        if ('\\' === DIRECTORY_SEPARATOR) {
```

Comment:
This is not really working, as the same behavior is expected also on linux/mac case-insensitive filesystems (yeah, macos is case-insensitive by default)